docs/docs/installation/start-developing.md

```diff
@@ -120,3 +120,6 @@ Sign in with the following credentials:
 `ADMIN_EMAIL` and `ADMIN_PASSWORD` can be set in `docker/.env` 
 :::
 
+:::note Note
+To avoid authentication issues during local development, you should set `AUTH_COOKIE_SECURE` to `False` in your `docker/.env`.
```

Comment:
Should we set `AUTH_COOKIE_SECURE=False` in `docker/env`?

Reply:
Yes, that's it! Just like `SECRET_KEY`, `ADMIN_EMAIL` and friends. Here is mine:

```
PRODUCTION=False
SECRET_KEY=XXX
ADMIN_EMAIL=admin@email.com
ADMIN_PASSWORD=admin
AUTH_COOKIE_SECURE=False
```

Reply:
Currently, the `Quickstart with Docker Compose` uses `docker-compose-non-dev.yaml` which uses the `docker/.env-non-dev` env file. Should we update `docker-compose-non-dev.yaml` to use `docker/.env` instead and include `AUTH_COOKIE_SECURE=False`?

This would allow first timers and those developing with Docker to use Safari.

Reply:
Well, actually it doesn't matter which file it's in. If they use `.env-non-dev`, then they should set in `.env-non-dev`.

That said, are you sure we need to have different `.env` files flavors? Couldn't we just use one?